arm64 support

has anybody gotten this to work an apple mac?

```
ustlib/x86_64-apple-darwin/lib" "-o" "/Users/tehlers/git/tui-journal/target/release/deps/tjournal-eccea6223eb12a8a" "-Wl,-dead_strip" "-nodefaultlibs"
  = note: ld: warning: ignoring duplicate libraries: '-lSystem'
          ld: warning: ignoring file '/opt/homebrew/Cellar/libgit2/1.7.2/lib/libgit2.1.7.2.dylib': found architecture 'arm64', required architecture 'x86_64'
          Undefined symbols for architecture x86_64:
```

--- a/Makefile
+++ b/Makefile
@@ -14,7 +14,10 @@ clippy:
 build-release:
 	cargo build --release
 
-release-mac: build-release
+build-release_mac_arm:
+	cargo build --release --target=aarch64-apple-darwin
+
+release-mac: build-release_mac_arm
 	strip target/release/tjournal
 	otool -L target/release/tjournal
 	mkdir -p release

--- a/src/app/ui/footer.rs
+++ b/src/app/ui/footer.rs
@@ -145,7 +145,7 @@ fn get_multi_select_text(ui_components: &UIComponents) -> String {
         .filter(|keymap| keymap.command == UICommand::ShowHelp)
         .collect();
 
-    let parts = vec![get_keymap_text(leave_keymap), get_keymap_text(help_keymap)];
+    let parts = [get_keymap_text(leave_keymap), get_keymap_text(help_keymap)];
 
     parts.join(SEPARATOR)
 }

